### DISCUSSION TITLE
Is this project active?

### DISCUSSION THREAD

**Discussion by ghost** [getgrav/grav#3663] [Help] [answered]
Hi all.

I want to help, but I want to know if this project is active. *Is this project active?*

**Comment by rhukster** [comment, marked as answer]
very active

Just not these forums.. check us out on discord or discourse: 

* https://chat.getgrav.org/
* https://discourse.getgrav.org/

**Reply by ghost** [reply]
Hi rhukster.

> very active
> Just not these forums.. check us out on discord or discourse:
> https://chat.getgrav.org/
> https://discourse.getgrav.org/

thank you for feedback.
